apps/website/src/pages/docs/footer.mdx

```diff
@@ -21,8 +21,6 @@ The footer is fixed to the bottom of a web page, and contains your typical legal
 
 The footer text is centered within the frame it is being displayed within, not centered within the viewport. This takes into consideration things that indent the body frame such as the side navigation.
 
-The hyperlinks within the footer are styled visibly different and use the $iui-text-color-muted color rather than the typical $iui-color-foreground-primary color. This is done purposefully to help the footer content not be confused with the actual content of the app.
-
 Optionally, you may include a 2nd line of text to display version numbers and/or server location for debugging reasons. You may choose to display this line to specific users such as beta users or Bentley Systems employees only.
 
 If you feel the need to add additional links, you may append links to the end of the footer’s first line.
```

Comment:
This one can be adjusted to not explicitly mention "first line"

Reply:
The change has been made